.github/workflows/release.yml

```diff
@@ -26,10 +26,10 @@ jobs:
 
       # we first have to clone the AddOn project, this is a required step
       - name: Clone project
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
           fetch-depth: 0  # gets git history for changelogs
 
       # once cloned, we just run the GitHub Action for the packager project
       - name: Package and release
-        uses: BigWigsMods/packager@v2.2.0
\ No newline at end of file
+        uses: BigWigsMods/packager@v2
```

Comment:
I pin to a specific version on purpose. Feel free to advance this, but keep this pinned to a specific semver.